tests/integration/preserve_ip_config_test.py

```diff
@@ -40,11 +40,11 @@
 def test_reapply_preserve_ip_config(eth1_up):
     libnmstate.apply(
         {
-            'interfaces': [
+            Interface.KEY: [
                 {
-                    'name': 'eth1',
-                    'type': 'ethernet',
-                    'state': 'up',
+                    Interface.NAME: 'eth1',
```

Comment:
I'm all good in fixing these too, but it seems you have not done this consistently in all files changed.
Whatever you choose, the commit message needs to reflect this.

Reply:
Done!